py-polars/docs/source/reference/config.rst

```diff
@@ -61,7 +61,7 @@ Note that ``Config`` supports setting context-scoped options. These options
 are valid *only* during scope lifetime, and are reset to their initial values
 (whatever they were before entering the new context) on scope exit.
 
-You can take advantage of this by initialising  a``Config`` instance and then
+You can take advantage of this by initialising a``Config`` instance and then
```

Comment:
Need a space in \`a\`\`Config\`\` (I think the one you deleted needed to be moved after the `a`).

Reply:
Yes. My mistake. Fixed it.